snow/engine/snowman/voter.go

```diff
@@ -55,13 +68,6 @@ func (v *voter) Update(ctx context.Context) {
 
 	for _, result := range results {
 		result := result
```

Comment:
do we still need this copy?

Reply:
This fixes a linter issue - after go 1.22(?) I don't think this will be needed.